src/index.html

```diff
@@ -22,6 +22,426 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#logo"
+        class="header__logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo company logo, showing the company name in blue and white"
+        />
+      </a>
+
+      <nav aria-label="Main Navigation">
+        <ul class="header__nav-menu">
+          <li>
+            <a
+              href="#apple"
+              class="header__nav-link header__nav-link--is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#samsung"
+              class="header__nav-link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#smartphones"
+              class="header__nav-link"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#laptops&amp;computers"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#gadgets"
+              class="header__nav-link"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#tablets"
+              class="header__nav-link"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#photo"
+              class="header__nav-link"
+            >
+              photo
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#video"
+              class="header__nav-link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image">
+          <img
+            src="images/imac.jpeg"
+            alt="iMac Product Image"
+            class="card__image"
+          />
+        </div>
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviev">Reviews: 5</p>
```

Comment:
There is a typo in your class name. It should be 'card__review' instead of 'card__reviev'.